src/app/App.js

```diff
@@ -17,19 +17,48 @@ export const App = ({options}) => {
    //add event listener to the select mode menu button 
    const help=document.querySelector('#helpScreen')
    const style = getComputedStyle(help);
+   let a=0
    document.querySelector('#whoIsThatPokemonOption').addEventListener('click',()=>{
       if(style.display=='none'||help.style.display == 'none')
          console.log("Who's that Pokemon?");
          SELECTED_MODE = WHO_IS_THAT_POKEMON;
+         
+         document.querySelector('#whatItLooksLikeOption').style.backgroundColor="#3762AC"
+         document.querySelector('#whatItLooksLikeOption').style.color="#FFCB05"
+         document.querySelector('#guessTheTypeOption').style.backgroundColor="#3762AC"
+         document.querySelector('#guessTheTypeOption').style.color="#FFCB05"
+         
+         document.querySelector('#whoIsThatPokemonOption').style.backgroundColor="#FFCB05"
+         document.querySelector('#whoIsThatPokemonOption').style.color="#3762AC" 
```

Comment:
you can extract it to function that looks something like:
```
const changeColorOfClickedButton = (querySelector) => {
  //first, get all 3 buttons
  //then, set all buttons styles to DEFAULT-COLORS
  //then, (based on querySelector), set "CLICKED-COLORS" to one, given button
}

```

and it should be one function that will take one of those 3 selectors you are using :)